src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -679,6 +679,12 @@ export default class ElementWrapper extends Wrapper {
 				`);
 			}
 		}
+
+		if (['button', 'input', 'keygen', 'select', 'textarea'].includes(this.node.name)) {
```

Comment:
What cases is this intended to handle? When would the `autofocus` attribute handling in the other file not be sufficient?

Reply:
From my understanding this deals with the case where `autofocus` is part of an attribute spread.

Reply:
This code handles when a spread attribute is used, and because we're not sure whether `autofocus` is true or not, this adds a more relaxed code: `if (node.autofocus) node.focus()`.

The other files however handle when an explicit `autofocus` prop is defined on an element and produces finer-grain code like: `node.focus()` or `if (bound_expression) node.focus()`.

An interesting behaviour I found was that if this code runs (spread attribute), the code in the other files doesn't, so there wouldn't be double `focus()` statements. (Unless I'm doing something wrong)